server/src/main/java/org/elasticsearch/search/fetch/subphase/highlight/AbstractHighlighterBuilder.java

```diff
@@ -206,6 +211,7 @@ public final void writeTo(StreamOutput out) throws IOException {
             out.writeMap(options);
         }
         out.writeOptionalBoolean(requireFieldMatch);
+        out.writeOptionalBoolean(limitToMaxAnalyzedOffset);
```

Comment:
you'll need to do a version check here so that mixed-cluster highlighting works

Reply:
Done it already, thx.